.travis.yml

```diff
@@ -18,3 +18,4 @@ install:
   - make -j2 V=
 script:
   - make -j2 test-ci
+  - make lint
```

Comment:
You can make it a separate task with `include`, an env var, and a conditional in `script` - it's probably worth doing it now.